pkg/pillar/types/zedroutertypes.go

```diff
@@ -1578,3 +1589,55 @@ type VpnMetrics struct {
 	PhyErrStat LinkPktStats
 	VpnConns   []*VpnConnMetrics
 }
+
+// IPTuple :
```

Comment:
I had to add these comments on each of these structure to satisfy yetus build checks.
@naiming-zededa Can you please add brief description for each of these structures?

Reply:
Sure. will add descriptions.